pkg/auth/authLogin.go

```diff
@@ -57,10 +58,15 @@ func GetJwtFromRestApi(apiServerUrl string, authProperties galasaapi.AuthPropert
 			err = galasaErrors.NewGalasaError(galasaErrors.GALASA_ERROR_RETRIEVING_BEARER_TOKEN_FROM_API_SERVER, err.Error())
 		} else {
 			defer httpResponse.Body.Close()
-			var tokenResponseJson []byte
-			tokenResponseJson, err = tokenResponse.MarshalJSON()
-			jwtJsonStr = string(tokenResponseJson)
-			log.Println("Bearer token received from API server OK")
+			var jwtJson []byte
+			token := utils.BearerTokenJson{
+				Jwt: tokenResponse.GetJwt(),
+			}
+			jwtJson, err = json.Marshal(token)
+			if err == nil {
+				jwtJsonStr = string(jwtJson)
+				log.Println("Bearer token received from API server OK")
+			}
```

Comment:
It would be worth adding a refresh token to the tests to make sure that only the JWT is stored